### PR TITLE
Align single node examples

### PR DESCRIPTION
**Pull request by jacobtomlinson**
We now have Single Node examples for AWS, Azure, GCP and IBM Cloud but they were all slightly different.

This PR is a refactor inspired from the best of each page to make them all consistent with each other.

I've also used some includes so that duplicated sections like pulling the Docker image and testing the RAPIDS install is only written once. 

I also tweaked the menus a bit.

### CODE REVIEW

File: source/_includes/menus/aws.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+````
+
+````{grid-item-card}
+:link: /cloud/aws/eks
+:link-type: doc
+Elastic Kubernetes Service (EKS)
+^^^
+Launch a RAPIDS cluster on managed Kubernetes.
+
+{bdg-primary}`multi-node`
```
Comment (mroeschke) [inline source/_includes/menus/aws.md:21]: Curious on your thoughts of making `multi-node` a different color than `single-node`

File: source/_includes/menus/gcp.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+`````{grid} 1 2 2 3
+:gutter: 2 2 2 2
+
+````{grid-item-card}
+:link: /cloud/gcp/compute-engine
+:link-type: doc
+Compute Engine Instance
+^^^
+Launch an Compute Engine instance and run RAPIDS.
```
Comment (mroeschke) [inline source/_includes/menus/gcp.md:9]:
```suggestion
Launch a Compute Engine instance and run RAPIDS.
```

**Comment by mroeschke** [comment]
Nice! Learned something new with the VMI for GCP. 

One grammar suggestion and a minor style thing otherwise looks good

File: source/_includes/menus/aws.md (hunk 1 above)
Comment (jacobtomlinson) [reply source/_includes/menus/aws.md:21]: Yeah I think that would be great. I want to think a bit more deeply about tagging in the docs, especially as we integrate these docs with cloud-ml-examples more closely. So maybe we could have this discussion then?

**Comment by jacobtomlinson** [comment]
Thanks for the review @mroeschke!

**Review by mroeschke — APPROVED** [review]

File: source/_includes/menus/aws.md (hunk 1 above)
Comment (mroeschke) [reply source/_includes/menus/aws.md:21]: Sounds good.

**Pull request merged**